.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+**PR Steps**
+
+These steps are to ensure you correctly make a PR. _Please delete them before submitting the PR!_
+
+1. For almost all development, you should be targeting `develop` rather than `main`. Only releases target `main`!
```

Comment:
Are we going to put something here about "make sure you run `make pr` before merging to ensure the PR matches linting" or similar? Or is that more on the general help pages I guess?